experimental/generation/generator/packages/library/templates/standard/en-us/dynamiclistEntity.en-us.lg.lg

```diff
@@ -0,0 +1,49 @@
+[standard.en-us.lg](standard.en-us.lg)
+
+# filename
+- ${entityLGFilename()}
+
+# template
+-```
+${formImport('form.en-us.lg')}
+${formImport('form-Choose.en-us.lg')}
+${propertyEntityValue()}
+${propertyEnumValue()}
+${chooseEntity()}
+```
+
+# chooseEntity
+-```
+# ${entity}_chooseEntity()
+> Choose between '${entity}' values
+- \${chooseEnumEntity()}```
+
+# propertyEntityValue
+-```
+# ${entity}_Value(entity) 
+> Display value for '${entity}'
+- \${choices(entity.value, '${enumName(entity)}', 'and')}```
```

Comment:
We should just return the entity.value.  The entity is open-ended so what we generate statically is not really useful.

Reply:
Instead of doing this, just call value(entit.value)

---
In reply to: [612049019](https://github.com/microsoft/BotBuilder-Samples/pull/3150#discussion_r612049019) [](ancestors = 612049019)